lib/fastly/dictionary.rb

```diff
@@ -0,0 +1,13 @@
+class Fastly
+  class Dictionary < BelongsToServiceAndVersion
+    attr_accessor :id, :name, :service_id
+
```

Comment:
How do you feel about adding some convenience methods here for adding/removing dictionary items?
e.g.

```
    def add_item(key, value)
      fetcher.create_dictionary_item(service_id: service_id, dictionary_id: id, item_key: key, item_value: value)
    end

    def delete_item(key)
      fetcher.delete_dictionary_item(service_id: service_id, dictionary_id: id, item_key: key)
    end
```